site/_scss/site/objects/_footer.scss

```diff
@@ -1,3 +1,18 @@
+@mixin breakpoint($point) {
```

Comment:
@amymanion you've already defined this mixin; remove and combine into a single mixins partial